src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsDiskMigrationJob.ps1

```diff
@@ -0,0 +1,155 @@
+
+.EXAMPLE
+
+    PS C:\> $disks = Get-AzsDisk -location local
+    PS C:\> New-AzsDiskMigrationJob -Name "MyMigrationJob" -Disks $disks -location local -TargetShare "\\SU1FileServer.azurestack.local\SU1_ObjStore"
+#>
+function New-AzsDiskMigrationJob
+{
+    [OutputType([Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob])]
+    [CmdletBinding(DefaultParameterSetName='DiskMigrationJobs_Create')]
+	[Alias('Start-AzsDiskMigrationJob')]
+    param(    
+        [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true, ParameterSetName = 'ResourceId_DiskMigrationJobs_Create')]
+        [System.String]
+        $ResourceId,
+    
+        [Parameter(Mandatory = $true, ParameterSetName = 'DiskMigrationJobs_Create')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'InputObject_DiskMigrationJobs_Create')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'ResourceId_DiskMigrationJobs_Create')]
+        [System.Array]
+        $Disks,
+    
+        [Parameter(Mandatory = $true, ParameterSetName = 'DiskMigrationJobs_Create')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'InputObject_DiskMigrationJobs_Create')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'ResourceId_DiskMigrationJobs_Create')]
+        [System.String]
+        $TargetShare,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'DiskMigrationJobs_Create')]
+        [System.String]
+        $Location,
+    
+        [Parameter(Mandatory = $true, ParameterSetName = 'DiskMigrationJobs_Create')]
+        [Alias('MigrationId')]
+        [System.String]
+        $Name,
+    
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true, ParameterSetName = 'InputObject_DiskMigrationJobs_Create')]
+        [Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob]
+        $InputObject
+    )
+
+    Begin 
+    {
+	    Initialize-PSSwaggerDependencies -Azure
+        $tracerObject = $null
+        if (('continue' -eq $DebugPreference) -or ('inquire' -eq $DebugPreference)) {
+            $oldDebugPreference = $global:DebugPreference
+			$global:DebugPreference = "continue"
+            $tracerObject = New-PSSwaggerClientTracing
+            Register-PSSwaggerClientTracing -TracerObject $tracerObject
+        }
+	}
+
+    Process {
+    
+    $ErrorActionPreference = 'Stop'
+
+    $NewServiceClient_params = @{
+        FullClientTypeName = 'Microsoft.AzureStack.Management.Compute.Admin.ComputeAdminClient'
+    }
+
+    $GlobalParameterHashtable = @{}
+    $NewServiceClient_params['GlobalParameterHashtable'] = $GlobalParameterHashtable
+     
+    $GlobalParameterHashtable['SubscriptionId'] = $null
+    if($PSBoundParameters.ContainsKey('SubscriptionId')) {
+        $GlobalParameterHashtable['SubscriptionId'] = $PSBoundParameters['SubscriptionId']
+    }
+
+    $ComputeAdminClient = New-ServiceClient @NewServiceClient_params
+
+    $MigrationId = $Name
```

Comment:
Remove

Reply:
We'd keep what it is look like because it is autogerated. And I don't think it has issue or has any user experience effect.